charts/router/values.yaml

```diff
@@ -5,3 +5,13 @@ platform_domain: ""
 dhparam: ""
 # limits_cpu: "100m"
 # limits_memory: "50Mi"
+
+# Any custom router annotations(https://github.com/deis/router#annotations)
+# which needs to be applied can be specified as key-value pairs under "deployment_annotations"
+deployment_annotations:
+  #<example-key>: <example-value>
+
+# Any custom annotations for k8s services like http://kubernetes.io/docs/user-guide/services/#ssl-support-on-aws
+# which needs to be applied can be specified as key-value pairs under "service_annotations"
```

Comment:
Same comment as above--since "annotations" is plural, the verb should be `need` not `needs`. Nitpick++. 😄